lib/util.js

```diff
@@ -291,6 +291,83 @@ exports.getVideoDescription = function(html) {
 
 
 /**
+ * Get video Owner from html
+ *
+ * @param {String} body
+ * @return {Object}
+ */
+exports.getOwner = function(body) {
+  var ownerinfo = this.between(body,'<div id="watch-header" class="yt-card yt-card-has-padding">','<div id="watch-action-panels" class="watch-action-panels yt-uix-button-panel hid yt-card yt-card-has-padding">')
+  var response = {};
+  if(ownerinfo != '') {
+    response.type = ownerinfo.match( new RegExp( '\/(user|channel)\/') )[1]
+    response.ref = ownerinfo.match( new RegExp('<a href="(\/'+response.type+'\/[\\w-]+)"') )[1]
+    response.id = response.ref.split('/')[2]
+    response.avatar = ownerinfo.match( new RegExp('<img.+?(?=data-thumb)data-thumb="([^"]+)"[^>]+>') )[1]
+    if(response.type == 'user') {
+      response.hiddenchannel = {};
+      response.hiddenchannel.type = 'channel'
+      response.hiddenchannel.ref = ownerinfo.match( new RegExp('<a href="(\/'+response.hiddenchannel.type+'\/[\\w-]+)"') )[1]
+      response.hiddenchannel.id = response.hiddenchannel.ref.split('/')[2]
+      response.hiddenchannel.name = ownerinfo.match( new RegExp('<a href="\/channel\/'+((response.type == 'user')?(response.hiddenchannel.id):(response.id))+'"[^>]+>(.+?(?=<\/a>))') )[1]
+    }
+    response.name = (response.type == 'user') ? response.hiddenchannel.name : ownerinfo.match( new RegExp('<a href="\/channel\/'+response.id+'"[^>]+>(.+?(?=<\/a>))') )[1]
+  }
+  return response;
+};
+
+
+/**
+ * Get video published at from html
+ *
+ * @param {String} body
+ * @return {String}
+ */
+exports.getPublished = function(body) {
+  return Date.parse( this.between(body,'<meta itemprop="datePublished" content="','">') )
+};
+
+
+/**
+ * Get video published at from html
+ *
+ * @param {String} body
+ * @return {Array.<Object>}
+ */
+//credits to https://github.com/paixaop
+exports.getRelatedVideos = function(body) {
+  jsonStr = this.between(body, 'yt.setConfig(', ');')
```

Comment:
If we're only interested in `RELATED_PLAYER_ARGS.rvs`, let's start our search there, from there til `"},` seems like it would work.

Add a semicolon at the end of the line.